lib/aiken/rats.ak

```diff
@@ -0,0 +1,88 @@
+// Rationals
+// Authors:
+// Micah Kendall
+// <other authors go here>
+
+pub type UncheckedRational {
+  numerator: Int,
+  denominator: Int,
+}
+
+pub opaque type Rational {
+  numerator: Int,
+  denominator: Int,
+}
+
+pub fn check_rational(num: UncheckedRational) -> Rational {
```

Comment:
This is the only place where 'UncheckedRational' is used. What about removing that type entirely and simply have that function takes two int arguments: numerator and denominator.

Reply:
It's for allowing a rational type in datums/redeemers while opaque times are still disallowed.